manifests/exports.pp

```diff
@@ -0,0 +1,16 @@
+# Class nfs::exports
```

Comment:
Please remove this from this PR, I'd like to have only one feature per PR (RedHat 7 support here)